### PR TITLE
update dependencies to latest with wdio@v7

### PR DESCRIPTION
**Pull request by wyhmichael**
Hi @fijijavis I made this PR to update the dependencies to latest to address some security alerts, please have a review on this, thanks!

### CODE REVIEW

**Comment by fijijavis** [comment]
@wyhmichael Thanks for the contribution! Running the builds and will publish if everything passes

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x]
+        node-version: [10.x, 12.x, 14.x]
```
Comment (fijijavis) [inline .github/workflows/nodejs.yml]:
```suggestion
        node-version: [12.x, 14.x]
```

**Comment by fijijavis** [comment]
v3.0.0 has been published to NPM

**Comment by wyhmichael** [comment]
Thanks @fijijavis for the fast response and update! I just figured I missed some changes, will make another PR for it

**Pull request merged**